capstone/backend/src/main/java/com/google/sps/spanner/UsersSpanner.java

```diff
@@ -0,0 +1,210 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.example.spanner;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.spanner.BackupId;
+import com.google.cloud.spanner.Database;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerException;
+import com.google.cloud.spanner.SpannerExceptionFactory;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.spanner.admin.database.v1.CreateDatabaseMetadata;
+import java.util.Arrays;
+import java.util.concurrent.ExecutionException;
+import org.threeten.bp.LocalDate;
+import org.threeten.bp.temporal.ChronoField;
+
+
+public class UsersSpanner {
+  static void createDatabase(DatabaseAdminClient dbAdminClient, DatabaseId id) {
+    OperationFuture<Database, CreateDatabaseMetadata> op =
+        dbAdminClient.createDatabase(
+            id.getInstanceId().getInstance(),
+            id.getDatabase(),
+            Arrays.asList(
+                "CREATE TABLE Users ("
+                    + "  UserId   INT64,"
+                    + "  Username  STRING(MAX),"
+                    + "  Email   STRING(MAX)"
+                    + ") PRIMARY KEY (UserId)",
+                "CREATE TABLE UserLists ("
+                    + "  UserId     INT64,"
+                    + "  ListId      INT64,"
+                    + "  ItemTypes   ARRAY<STRING(MAX)>"
+                    + ") PRIMARY KEY (UserId, ListId),"
+                    + "  INTERLEAVE IN PARENT Users ON DELETE CASCADE",
+                "CREATE TABLE Stores ("
+                    + "StoreId      INT64,"
+                    + "StoreName    STRING(MAX),"
+                    + "Address     STRING(MAX)"
+                    + ") PRIMARY KEY (StoreId)",
+                "CREATE TABLE Items ("
+                    + "ItemId      INT64,"
+                    + "ItemNameAndBrand    STRING(MAX),"
+                    + "ItemType     STRING(MAX)"
+                    + ") PRIMARY KEY (ItemId)",
+                "CREATE TABLE Inventory ("
+                    + "StoreId     INT64,"
+                    + "ItemId      INT64,"
+                    + "ItemAvailability STRING(MAX),"
+                    + "LastUpdated TIMESTAMP OPTIONS (allow_commit_timestamp=true),"
+                    + "Price FLOAT64,"
```

Comment:
The formatting here is weird: the spaces between the variable and the type is inconsistent